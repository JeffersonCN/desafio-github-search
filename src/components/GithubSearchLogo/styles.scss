@font-face {
    font-family: 'Monaco';
    src: local('Monaco'), url(./assets/fonts/monaco-webfont.woff) format('woff');
}

.githubSearchLogo {
    font-size: 60px;
    color: #000;

    .githubText {
        font-family: 'Monaco', sans-serif;
    }

    .searchText {
        font-family: 'Raleway';
        font-weight: 200;
        font-style: italic;
    }
}