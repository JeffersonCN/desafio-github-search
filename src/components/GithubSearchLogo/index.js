import React, { Component } from 'react';

import './styles.scss';

class GithubSearchLogo extends Component {
    render() {
        return (
            <div className="github-search-logo" >
                <span className="github-text" >Github</span>
                <span className="search-text" > Search</span>
            </div>
        );
    }
}

export default GithubSearchLogo;