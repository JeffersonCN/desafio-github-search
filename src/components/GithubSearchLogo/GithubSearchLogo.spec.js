import React from 'react';
import { shallow } from 'enzyme';
import GithubSearchLogo from './';

describe('Testing GithubSearchLogo Component', () => {
    it('shoud render correctly', () => {
        const wrapper = shallow(
            <GithubSearchLogo />
        );

        expect(wrapper).toMatchSnapshot();
    });
});
