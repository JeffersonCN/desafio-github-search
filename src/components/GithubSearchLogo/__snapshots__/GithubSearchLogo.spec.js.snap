// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing GithubSearchLogo Component shoud render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <GithubSearchLogo />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <span
          className={undefined}
>
          Github
</span>,
        <span
          className={undefined}
>
           Search
</span>,
      ],
      "className": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Github",
          "className": undefined,
        },
        "ref": null,
        "rendered": "Github",
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " Search",
          "className": undefined,
        },
        "ref": null,
        "rendered": " Search",
        "type": "span",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <span
            className={undefined}
>
            Github
</span>,
          <span
            className={undefined}
>
             Search
</span>,
        ],
        "className": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Github",
            "className": undefined,
          },
          "ref": null,
          "rendered": "Github",
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " Search",
            "className": undefined,
          },
          "ref": null,
          "rendered": " Search",
          "type": "span",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
